packages/e2e/features/ui/components/storage/storage-browser/search-locations.feature

```diff
@@ -0,0 +1,47 @@
+Feature: Search with Storage Browser
+
+    Background:
+        Given I'm running the example "ui/components/storage/storage-browser/default-auth"
+
+    @react
+    Scenario: Basic search returns correct results
+    When I type my "email" with status "CONFIRMED"
+    Then I type my password
+    Then I click the "Sign in" button
+    When I click the first button containing "public"
+    Then I see the button containing "test+name"
+    When I see input with placeholder "Search current folder" and type "DO_NOT"
+    Then I click the "Search" button
+    Then I see the button containing "DO_NOT_DELETE"
+    Then I do not see the button containing "test+name"
+    # Verify clear returns to initial state
+    When I click the button containing "Clear search"
+    Then I see the button containing "test+name"
+
+    @react
```

Comment:
nit: scenario one and two can be combined into one for me. I see `DO_NOT_DELETE` but not `DO_NOT_DELETE/DONT_DELETE_SUB` and clicking include sub folders covers both